### PR TITLE
Update colab for new viewer

### PR DESCRIPTION
**Pull request by ginazhouhuiwu**
Changes:
- deprecate old viewer elements in notebook
- use conda for colmap installation

TODO: Viser share link doesn't seem to be printing properly. `ns-train` cell sometimes stops executing after saving the configs.

### CODE REVIEW

File: colab/demo.ipynb [hunk 1]
```diff
@@ -58,59 +61,57 @@
...
       "execution_count": null,
       "metadata": {
-        "id": "9oyLHl8QfYwP",
-        "cellView": "form"
+        "cellView": "form",
+        "id": "9oyLHl8QfYwP"
       },
       "outputs": [],
       "source": [
-        "#@markdown <h1>Install Nerfstudio and Dependencies (~8 min)</h1>\n",
```
Comment (akristoffersen) [inline colab/demo.ipynb:73]: Does this still take around 8min?

**Comment by yimingc** [comment]
Thanks for trying fix this.

I got error when I ran in colab on your branch tip
image

**Comment by ginazhouhuiwu** [comment]
Hey! Thanks for letting me know the errors! Unfortunately I'm still working on debugging the training portion in this pr draft... But your error seems to point to a tinycudann installation error, which I can't recreate from my end. Can you let me know if that was properly installed from running the first cell in the notebook?

File: colab/demo.ipynb (hunk 1 above)
Comment (ginazhouhuiwu) [reply colab/demo.ipynb:73]: Will check, it took kind of long for me.

**Comment by AntonioMacaronio** [comment]
This is still an error - on line 14 inside the cell "Install Nerfstudio and Dependencies (~8 min)":
`!gdown "https://drive.google.com/u/0/uc?id=15WngFRNar_b8CaPR5R-hvQ3eAnlyk_SL&confirm=t"`

tinycudann is able to be installed perfectly, but the drive link for the colmap package is not sharable (even with the `confirm=t` tag on the url for the gdrive file). Perhaps that is why it is not reproducible from your end @ginazhouhuiwu 

to fix this, maybe all that's needed is an updated and sharable gdrive link?

File: colab/demo.ipynb [hunk 2]
```diff
@@ -58,59 +61,57 @@
...
+        "#@markdown <h3>Install Nerfstudio and Dependencies (~8 min)</h3>\n",
         "\n",
         "%cd /content/\n",
         "!pip install --upgrade pip\n",
         "!pip install torch==2.0.1+cu118 torchvision==0.15.2+cu118 --extra-index-url https://download.pytorch.org/whl/cu118\n",
```
Comment (Lothiraldan) [inline colab/demo.ipynb:87]:
```suggestion
        "!pip install torch==2.0.1+cu118 torchvision==0.15.2+cu118 torchaudio==2.0.2+cu118  --extra-index-url https://download.pytorch.org/whl/cu118\n",
```

I also needed to pin the version of torchaudio to get training to start on Google Colab. I only tested it on Google Colab and as of today.

**Pull request closed without merging**